croncheck.php

```diff
@@ -32,6 +32,8 @@
 
 define('NO_UPGRADE_CHECK', true);
 
+include('iplock.php');
```

Comment:
this is fine as is, but it's probably a better habit to always use require_once() for things which are libs and it's also more 'moodley'